--- origsrc/gtksourceview-3.18.1/data/glade/gtksourceview.xml.in	2015-10-03 09:56:51.000000000 -0500
+++ src/gtksourceview-3.18.1/data/glade/gtksourceview.xml.in	2015-11-18 17:06:53.494367200 -0600
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<glade-catalog version="3.0" supports="gtkbuilder" name="gtksourceview" library="gtksourceview-3.0" domain="gtksourceview-3.0" depends="gtk+" book="gtksourceview-3.0">
+<glade-catalog version="3.0" supports="gtkbuilder" name="gtksourceview" library="gtksourceview-3.0-1" domain="gtksourceview-3.0" depends="gtk+" book="gtksourceview-3.0">
   <glade-widget-classes>
     <glade-widget-class name="GtkSourceView" title="GtkSourceView" generic-name="gtksourceview">
       <properties>
