--- a/data/glade/gtksourceview.xml.in
+++ b/data/glade/gtksourceview.xml.in
@@ -2,7 +2,7 @@
 <glade-catalog version="3.0"
                supports="gtkbuilder"
                name="gtksourceview"
-               library="gtksourceview-@GSV_API_VERSION@"
+               library="gtksourceview-@GSV_API_VERSION@-1"
                domain="gtksourceview-@GSV_API_VERSION@"
                depends="gtk+"
                book="gtksourceview-@GSV_API_VERSION@">
